### PR TITLE
Remove save button

### PR DESCRIPTION
**Pull request by pphillips-fearless**
Remove the save button

### This pull request changes...

- the component is removed
- other components that exist solely to support the floating save button are removed (the only thing I could find is getSaveApdWorking)
- get rid of the old redux state/selectors that the floating save button used

### This pull request is ready to merge when...

- [ ] Tests have been updated (and all tests are passing)
- [ ] This code has been reviewed by someone other than the original author
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented
- [ ] The change has been documented
  - [ ] Associated OpenAPI documentation has been updated
- [ ] Changelog is updated as appropriate

### This feature is done when...

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: web/src/reducers/apd.js [hunk 1]
```diff
@@ -369,23 +368,6 @@ const reducer = (state = initialState, action) => {
...
-            ...state.byId[action.data.id],
-            updated: getHumanTimestamp(action.data.updated)
-          }
-        },
-        data: {
-          ...state.data,
-          created: getHumanDatestamp(action.data.created),
-          updated: getHumanTimestamp(action.data.updated)
-        }
-      };
```
Comment (mgwalker) [inline web/src/reducers/apd.js]: I don't think we can get rid of this. Without it, the APD "last updated" time won't ever get updated. We use the APD state's `updated` property on the state dashboard (that's how we refer to the current landing page).

File: web/src/reducers/working.js [hunk 2]
```diff
@@ -72,13 +72,11 @@ const reducer = (state = initialState, { type }) => {
...
 const getEditOwnAccountWorking = state => state.working.editOwnAccount;
-const getSaveApdWorking = state => state.working.saveApd;
 
 export default reducer;
 
 export {
   getAddAccountWorking,
   getEditAccountWorking,
-  getEditOwnAccountWorking,
-  getSaveApdWorking
```
Comment (mgwalker) [inline web/src/reducers/working.js]: Oh, nice catch! I think there's more in this file that we should remove too. There should be some maps that link action symbols to working state properties. I think the `SAVE_APD_*` actions are in those maps, and those can come out. There is a switch below that can be updated too.

File: web/src/reducers/apd.test.js [hunk 3]
```diff
@@ -1049,54 +1048,6 @@ describe('APD reducer', () => {
...
-        },
-        data: {
-          created: 'July 30, 1965',
-          name: 'Timmothert',
-          updated: 'April 11, 1953, 12:00 AM GMT'
-        }
-      },
-      { type: SAVE_APD_SUCCESS, data: { id: 'apdID', updated: '' } }
-    );
-  });
```
Comment (mgwalker) [inline web/src/reducers/apd.test.js]: Only last thing is we should also put these tests back.

Comment (pphillips-fearless) [reply web/src/reducers/apd.test.js]: oh, good catch, I forgot that one!

**Pull request merged**